### PR TITLE
Add koleaje olayinka to BID

### PR DESCRIPTION
**Pull request by koleajeolayinka**
I'm adding my data to the BID

### CODE REVIEW

File: CONTRIBUTORS.md
Comment (XanderRubio) [inline CONTRIBUTORS.md:1]: Hi @koleajeolayinka! I made a small change to ensure the formatting shows up nicely when viewing the CONTRIBUTORS.md file. You did everything correctly and when you click on your avatar it takes you to your GitHub account.

File: src/Jsons/Contributors.json
Comment (XanderRubio) [inline src/Jsons/Contributors.json:1]: I added a `.` to the end of your `"text"`. You did everything correctly ✅

**Review by XanderRubio — APPROVED** [review]
Hi @koleajeolayinka,

Thank you for taking the time to contribute your image and text for what you want to do before you die! Congratulations on being the first contributor from Nigeria also! Your image and text look great on the preview deployment, and I will merge your work in the main now. Improving oneself daily and helping others is a life full of intention and rewards, which is a great way to live.

**Comment by XanderRubio** [comment]
@koleajeolayinka   Your pull request has been successfully merged!!! Congratulations Bottle with Popping CorkBottle with Popping CorkBottle with Popping CorkDouble Exclamation MarkDouble Exclamation MarkDouble Exclamation Mark. Make it happen for living a life of constant improvement and helping others!

 If you haven't already, please go ahead and star Glowing StarGlowing StarGlowing Starthis repo and you are more than welcome to continually contribute to this project by assisting with our [open issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues), or adding your own new open issues, and additionally you can get an idea of ideas we have to continually grow this project further as technology and tech stack continually evolve by looking at the [ROADMAP](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/ROADMAP.md) for the project. Have a great day, and thank you for taking the time to contribute, Koleaje! 
 
 You can view your contribution now on the [live link](https://before-i-die-achievements.vercel.app/). Have a great rest of your day, and I will be sending you an invite to be a member of the Before I Die Code now.
 
 Xander

**Comment by koleajeolayinka** [comment]
Hi @XanderRubio ,

Thank you so much! It means a lot that you appreciate my little contribution. I'm excited to be part of this and can't wait to contribute more. 💥

**Comment by XanderRubio** [comment]
> Hi @XanderRubio ,
> 
> Thank you so much! It means a lot that you appreciate my little contribution. I'm excited to be part of this and can't wait to contribute more. 💥

My pleasure @koleajeolayinka, and keep improving daily as this is what we can always control. Have a great rest of your day!

**Pull request merged**